index.src.html

```diff
@@ -1477,41 +1481,24 @@
         adjustments.
   </dd>
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropLeft</dfn></dt>
-  <dd>
-    The number of pixels to remove from the left of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>cropTop</dfn></dt>
-  <dd>
-    The number of pixels to remove from the top of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
+  NOTE: {{VideoDecoderConfig/codedWidth}} and {{VideoDecoderConfig/codedHeight}}
+    are used as hints when selecting a {{VideoDecoder/[[codec implementation]]}}.
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropWidth</dfn></dt>
+  <dt><dfn dict-member for=VideoDecoderConfig>displayAspectWidth</dfn></dt>
   <dd>
-    The width in pixels to include in the crop, starting from cropLeft.
-        Defaults to codedWidth if not present.
+    Horizontal component of the VideoFrame's aspect ratio when displayed.
```

Comment:
Maybe use another word than "component", as this is the term used for something else ? Maybe simply "dimension" ?

Reply:
Seems good, updated.